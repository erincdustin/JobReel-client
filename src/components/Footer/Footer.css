.footer {
  max-width: 100vw;
  background: #42836e;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
  grid-gap: 16px;
  height: 75px;
}

.team-info {
  grid-column-start: 2;
  grid-column-end: 4;
  margin: auto;
}

.links-to-tech {
  grid-column-start: 11;
  grid-column-end: 13;
  margin: auto;
}


