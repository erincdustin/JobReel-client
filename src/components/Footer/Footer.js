import React from 'react';
import './Footer.css'


function Footer(props) {
  
    return(
      <footer className='footer'>
        <div className='team-info'>
          Created by:
          Aldiyar Batyrbekov
          Chris Fiander
          Erin Dustin
          Jonathan Moore
          Michael Romero
        </div>
        <div className='links-to-tech'>
          <a href='https://github.com/thinkful-ei-dragonfly/JobReel-client' rel='noopener noreferrer' target='_blank'>GitHub Client</a>
          <br/>
          <a href='https://github.com/thinkful-ei-dragonfly/JobReel-server' rel='noopener noreferrer' target='_blank'>GitHub Server</a>
        </div>
      </footer>
    )
}

export default Footer;