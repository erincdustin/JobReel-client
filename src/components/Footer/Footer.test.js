import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import {BrowserRouter} from 'react-router-dom'
import Footer from './Footer'

describe('Footer smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Footer /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('Footer snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})