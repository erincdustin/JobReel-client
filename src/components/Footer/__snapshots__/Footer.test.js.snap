// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test renders the UI as expected 1`] = `
<footer
  className="footer"
>
  <div
    className="frontEnd"
  >
    <a
      href="https://github.com/thinkful-ei-dragonfly/JobReel-client"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub Client
    </a>
  </div>
  <div
    className="contactUs"
  >
    <Link
      to="/communicate"
    >
      Contact Us
    </Link>
  </div>
  <div
    className="backEnd"
  >
    <a
      href="https://github.com/thinkful-ei-dragonfly/JobReel-server"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub Server
    </a>
  </div>
</footer>
`;
